test/check-storage-reclaim

```diff
@@ -216,6 +216,30 @@ class TestStorageUseFreeSpaceScenarioExistingSystemFedora(anacondalib.VirtInstal
         i.reach(i.steps.REVIEW)
         r.check_affected_system("Fedora Linux", [("vda4", ["home", "root", "var"])])
 
+@nondestructive
+class TestStorageUseFreeSpaceScenarioExistingSystemDebian(anacondalib.VirtInstallMachineCase, StorageCase):
```

Comment:
This may go in a separate commit but I don't insist, just making sure it is really intended to go in.